engine/src/main/scala/cromwell/webservice/ApiDataModels.scala

```diff
@@ -63,6 +65,8 @@ object APIResponse {
 
   /** When a request completes successfully. */
   def success(message: String, data: Option[JsValue] = None) = SuccessResponse("success", message, data)
+
+  def workflowNotFound(id: WorkflowId) = RequestComplete(StatusCodes.NotFound, APIResponse.error(new Throwable(s"Workflow '$id' not found.")))
```

Comment:
Instead of using the base class `new Throwbable`, use `new RuntimeException` if you're feeling java-esque, or even `new Exception`.